src/main/java/com/airsquared/blobsaver/Main.java

```diff
@@ -18,26 +18,37 @@
 
 package com.airsquared.blobsaver;
 
+import com.airsquared.blobsaver.model.Version;
 import com.sun.javafx.PlatformUtil;
-import com.sun.javafx.application.PlatformImpl;
+import de.codecentric.centerdevice.util.StageUtils;
 import it.sauronsoftware.junique.AlreadyLockedException;
 import it.sauronsoftware.junique.JUnique;
 import javafx.application.Application;
 import javafx.application.Platform;
 import javafx.fxml.FXMLLoader;
+import javafx.geometry.Rectangle2D;
 import javafx.scene.Parent;
 import javafx.scene.Scene;
 import javafx.scene.image.Image;
+import javafx.stage.Screen;
 import javafx.stage.Stage;
+import me.matetoes.libdockvisibility.DockVisibility;
 
 import java.io.IOException;
+import java.net.URL;
+import java.util.Objects;
 import java.util.prefs.Preferences;
 
 public class Main {
 
-    static final String appVersion = "v2.2.2";
+    //enables debug menu icon in tray icon and to create a breakpoint.
+    //You can also create a breakpoint on the helpLabelHandler() function and click the question marks to debug
+    static final boolean DEBUG_MODE = false;
```

Comment:
Could you rename this variable to something else, as it may be confusing(debug mode could mean extra log output which isn't the case)

Reply:
What should I rename it to? I don't know anything about the log output stuff. The comments say pretty clear what it is though.

Reply:
Uh, maybe `SHOW_BREAKPOINT_OPTION` or something similar?